drizzlepac/haputils/mvm_quality_analysis.py

```diff
@@ -390,11 +390,6 @@ def overlap_crossmatch_analysis(total_obj_list, sourcelist_type="point", good_fl
                 log.warning("Unable to locate one or more SVM sourcelist(s) required for crossmatch!")
```

Comment:
The two datasets I use for processing fail at line 389.  The problem is 
`    if overlap_dict[bit_value]["svm_sourcelist_{}".format(set_num)] is None:
KeyError: 'svm_sourcelist_0'
`
Since I pulled the SVM FLT/FLC and catalogs to my local directory, is there something else needed?

Reply:
Issue _should_ have been resolved by commit [ebe4f8e](https://github.com/spacetelescope/drizzlepac/pull/1413/commits/ebe4f8ec593c3ddc1e4e56c56c0cb2043a31c8d2)